### PR TITLE
fix: simplify variable initialization

### PR DESCRIPTION
**Pull request by thebedigupta**
**Description**

In this PR I try to solve this [issue](https://sonarcloud.io/project/issues?impactSeverities=LOW&issueStatuses=OPEN%2CCONFIRMED&id=asyncapi_generator&open=AZB4sei0gviDxNeZYbKO)

I am also attching test files captured results [test-results.txt](https://github.com/user-attachments/files/17263501/test-results.txt)

- unnecessary initialization

### CODE REVIEW

**Comment by Light13008** [comment]
mb, i will fix this in some time.

File: apps/generator/cli.js
Comment (DhairyaMajmudar) [inline apps/generator/cli.js:1]: Don't delete this file, Please get it back

**Comment by thebedigupta** [comment]
> remove changes to cli.js and as usual make sure you do not use any autoformatter

done

**Review by derberg — COMMENTED** [review]
`cli.js` is still showing up in the list of modified files

**Comment by thebedigupta** [comment]
> `cli.js` is still showing up in the list of modified files

let me create another fresh PR for this issue

**Comment by DhairyaMajmudar** [comment]
> let me create another PR for this issue

@thebedigupta no need to open a separate PR, by mistake, you've pushed the empty `cli.js` file since the content in the file is removed that's why it's showing in the list of modified files. To fix this you can follow the following steps:

-> Copy and paste the content of the original [cli.js](https://github.com/asyncapi/generator/blob/master/apps/generator/cli.js) file to your empty file.
-> Commit and push the changes to your PR

**Comment by thebedigupta** [comment]
> > let me create another PR for this issue
> 
> @thebedigupta no need to open a separate PR, by mistake, you've pushed the empty `cli.js` file since the content in the file is removed that's why it's showing in the list of modified files. To fix this you can follow the following steps:
> 
> -> Copy and paste the content of the original [cli.js](https://github.com/asyncapi/generator/blob/master/apps/generator/cli.js) file to your empty file. -> Commit and push the changes to your PR

Oh it's that much easy ok I will do it and thank you for your suggestion

**Comment by thebedigupta** [comment]
> > > let me create another PR for this issue
> > 
> > 
> > @thebedigupta no need to open a separate PR, by mistake, you've pushed the empty `cli.js` file since the content in the file is removed that's why it's showing in the list of modified files. To fix this you can follow the following steps:
> > -> Copy and paste the content of the original [cli.js](https://github.com/asyncapi/generator/blob/master/apps/generator/cli.js) file to your empty file. -> Commit and push the changes to your PR
> 
> Oh it's that much easy ok I will do it and thank you for your suggestion

I tried it but it was not working i don't know why anyway let me create another PR for this removing these `cli.js` file issues

**Pull request closed without merging**